### ISSUE TITLE
Sharing public album button don't work

### ISSUE THREAD

**Issue by Hipisu** [pulsejet/memories#384]
Hello,

Im using the Memories 4.10.3 with Nextcloud Hub 3 (25.0.2) [PHP 8.0.25], When I click the public album share button nothing happens. I was only able to indentify the following error in the console:

![obraz](https://user-images.githubusercontent.com/123890605/215340633-bd4fd848-c2c5-4758-a81b-c928c4b4b31f.png)

Can you please assist? Thank you very much :)

**Comment by EVOTk** [comment]
Hello,
We have a very similar configuration ( NC 25.0.2 / PHP8.0.25 ). And I don't have this problem. 

What browser (+version) are you using?
Do you use a reverse proxy?

**Comment by pulsejet** [comment]
Also, is this incognito? I don't believe the clipboard API works right there.

**Comment by Hipisu** [comment]
Firefox 109.0.1
Chrome 109.0.5414.120

on both browsers issue looks the same, I as well checked incognito mode and it makes no difference.

Im not utilizing reverse proxy - its just small private instance based on Snap on Ubuntu.

**Comment by pulsejet** [comment]
Just to be sure: are you running Nextcloud over HTTPS?

**Comment by pulsejet** [comment]
Closing the issue in that case. For anyone coming to this later,

tl;dr you need HTTPS for the clipboard API to work (and many others too).
